src/main/scala/cognite/spark/v1/DefaultSource.scala

```diff
@@ -193,6 +193,10 @@ class DefaultSource
         new TimeSeriesRelation(config)(sqlContext)
       case "assets" =>
         new AssetsRelation(config)(sqlContext)
+      case "datapoints" =>
+        new NumericDataPointsRelationV1(config)(sqlContext)
+      case "string_datapoints" =>
```

Comment:
We call this `stringdatapoints`. Since we're using those strings in multiple places (and to avoid errors like this) we should probably move them to Constants and define them once instead.